internal/pkg/cli/pipeline_init.go

```diff
@@ -290,12 +292,16 @@ func (o *initPipelineOpts) askCodeCommitRepoDetails() error {
 	o.repoName = repoDetails.name
 	o.ccRegion = repoDetails.region
 
-	// If any one of the chosen environments is in a region besides that of the CodeCommit repo, pipeline init errors out.
-	for _, env := range o.envConfigs {
-		if env.Region != o.ccRegion {
-			return fmt.Errorf("repository %s is in %s, but environment %s is in %s; they must be in the same region", o.repoName, o.ccRegion, env.Name, env.Region)
-		}
+	// If the CodeCommit region is different than that of the app, pipeline init errors out. TODO: compare account from app config against CC acct?
+	sess, err := o.sessProvider.Default()
+	if err != nil {
+		return fmt.Errorf("retrieve session: %w", err)
 	}
+	region := *sess.Config.Region
```

Comment:
Is this guaranteed to be a non-nil pointer? 
```suggestion
	region := aws.StringValue(sess.Config.Region)
```

Reply:
Fixed in previous commit! 😄